containerz/containerz.proto

```diff
@@ -37,18 +37,34 @@ option (types.gnoi_version) = "0.1.2";
 
 service Containerz {
   // Deploy sets a container image on the target. The container is sent as
-  // a sequential stream of messages containing up to 64KB of data. Upon
-  // reception of a valid container, the target must load it into its registry.
-  // Whether the registry is local or remote is target and deployment specific.
-  // A valid container is one that has passed its checksum.
+  // a sequential stream of messages containing up to 4MB of data (max grpc
```

Comment:
note that 4MB is the default max. value, but it is configurable. hence, this phrasing is ambiguous.